account_invoice_discount_display_amount/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,4 @@
+* Manuel Regidor <manuel.regidor@sygel.es>
+* Alexei Rivera <arivera@archeti.com>
+- [APSL](https://apsl.tech):
+  - Santi Amorós \<<samoros@apsl.net>\>
```

Comment:
Incorrect RST syntax. This is RST, not Markdown (you can convert all to MD, but not mix them, and more having .rst extension).

Reply:
Fix it!